tox.ini

```diff
@@ -125,7 +125,6 @@ changedir =
 
 commands_pre =
 ; Install without -e to test the actual installation
-  py3{6,7,8,9}: python -m pip install -U pip setuptools wheel
```

Comment:
i thought this was necessary to ensure the latest version of those packages are used